src/language/walker/ruleWalker.ts

```diff
@@ -57,11 +57,7 @@ export class RuleWalker extends SyntaxWalker implements IWalker {
     }
 
     public hasOption(option: string): boolean {
-        if (this.options !== undefined) {
-            return this.options.indexOf(option) !== -1;
-        } else {
-            return false;
-        }
+        return this.options !== undefined ? this.options.indexOf(option) !== -1 : false;
```

Comment:
theres no need for the conditional expression, just use `return this.options !== undefined && this.options.indexOf(option) !== -1`